main.go

```diff
@@ -24,35 +24,42 @@ var (
 
 func init() {
 	// loads values from .env into the system
-	if err := godotenv.Load(); err != nil {
-		log.Print("No .env file found")
-	}
-	userName, exists := os.LookupEnv("USERNAMEEMAIL")
-	if exists {
+	if err := godotenv.Load(); err == nil {
+		userName, exists := os.LookupEnv("USERNAMEEMAIL")
+		if !exists {
+			log.Fatalln(".env should contain USERNAMEEMAIL")
```

Comment:
Clearer message, since its not always using .env file

`log.Fatalln("Environment variable should contain USERNAMEEMAIL")`

Reply:
updated @Izzur, please check again